utils/lib_govm.sh

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# Go versions manager to install and maintain golang [1] binaries & packages.
+#
+# [1] https://golang.org/doc/devel/release#policy
+
+# shellcheck source=utils/lib.sh
+. /dev/null
+
+# configure golang environment for go.vm
+# --------------------------------------
+
+_GO_DL_URL="https://go.dev/dl"
+
+GOVERSION="${GOVERSION:-go1.24.5}"
```

Comment:
We should probably pick the version defined in `go.mod`

```suggestion
GOVERSION="${GOVERSION:-go$(awk '/^go /{print $2}' "${REPO_ROOT}/go.mod")}"
```

I don't know about Dependabot (probably the same thing), but [Renovate](https://github.com/renovatebot/renovate) bumps the minimum version when some module requires a higher version than the one defined in `go.mod`

Reply:
Good catch .. [diff](https://github.com/searxng/searxng/compare/cbaef5ab8e8bbc89dc396e3386db7705970ac10b..b26c7d9f5afc66fe6652bdbb0ab43bde89f0acec)